.travis.yml

```diff
@@ -10,3 +10,5 @@ script:
 - (test $TRAVIS_RUST_VERSION != "nightly" || (cd phf_macros && cargo bench))
 - (cd phf_codegen && cargo test)
 - (cd phf_codegen/test && cargo test)
+- (cd phf_shared && cargo build --features core)
```

Comment:
These will need to be flagged in the same way as the phf_macros tests are above since no_std is unstable.